### PR TITLE
add a hypothetical embed script to be included with the oembed endpoint

### PR DESCRIPTION
**Pull request by jamesgorrie**
Related to #2786 

When combined, the powers of these PRs should allow us to start testing some sort of oEmbed.

### CODE REVIEW

File: client/embed.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+(function() {
+  var wecoWorkEmbeds = document.querySelectorAll('.weco-work-embed');
```
Comment (davidpmccormick) [inline client/embed.js]: Out of interest, why `var`s not `const`s?

File: client/embed.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  wecoWorkEmbeds.forEach(el => {
+    if (!el.getAttribute('data-weco-enhanced')) {
+      var workId = el.getAttribute('data-weco-work-id');
+      var parentEl = el.parentElement;
+      el.setAttribute('data-weco-enhanced', 'true');
+      el.addEventListener('click', function() {
```
Comment (davidpmccormick) [inline client/embed.js]: …and why not an arrow function?

Comment (jamesgorrie) [reply client/embed.js]: In browser - although it is being compiled.

File: client/embed.js (hunk 1 above)
Comment (jamesgorrie) [reply client/embed.js]: In browser - although it is being compiled.

**Pull request merged**